editing.py

```diff
@@ -181,36 +181,40 @@ def add_area(self, area, edit_note, auto=False):
 
         data_template = {
             "edit-area.name": area["name"],
+            # Why is disambiguation named "comment" on MB?
```

Comment:
That used to be the old name of the field :) it's a "disambiguation comment" but I guess it was renamed to disambiguation because people were using it to write random comments? Dunno, it's an old thing :) But renaming the stuff in the database schema is not trivial and we haven't had a big reason to.

Reply:
But is there a reason someone using the bot API has to use "disambiguation" as the key?

I'm asking because if the keys in `area` matched the field names 1:1, we wouldn't need `data_template` at all. We could just do `create_payload('edit-area', area, required_fields)`.